Source/Android/app/build.gradle

```diff
@@ -64,8 +64,8 @@ android {
     defaultConfig {
         externalNativeBuild {
             cmake {
-                arguments "-DANDROID_STL=c++_static", "-DCMAKE_BUILD_TYPE=RelWithDebInfo" // , "-DENABLE_GENERIC=ON"
-                abiFilters "arm64-v8a" //, "armeabi-v7a", "x86_64", "x86"
+                arguments "-DANDROID_STL=c++_static", "-DCMAKE_BUILD_TYPE=RelWithDebInfo", "-DENABLE_GENERIC=ON"
```

Comment:
I changed these to allow the emulator to run on my phone. Otherwise when I tried to build and run it I would get errors. I'll change these back.